daemon/rpc_servers.go

```diff
@@ -0,0 +1,81 @@
+package daemon
+
+import (
+	"context"
+	"slices"
+
+	"github.com/NordSecurity/nordvpn-linux/config"
+	"github.com/NordSecurity/nordvpn-linux/core"
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
+)
+
+func techToProto(tech core.ServerTechnology) pb.Technology {
+	switch tech {
+	case core.OpenVPNUDP:
+		return pb.Technology_OPENVPN_UDP
+	case core.OpenVPNTCP:
+		return pb.Technology_OPENVPN_TCP
+	case core.OpenVPNUDPObfuscated:
+		return pb.Technology_OBFUSCATED_OPENVPN_UDP
+	case core.OpenVPNTCPObfuscated:
+		return pb.Technology_OBFUSCATED_OPENVPN_TCP
+	case core.WireguardTech:
+		return pb.Technology_NORDLYNX
+	}
+
+	return pb.Technology_UNKNWON
+}
+
+func groupFilter(groups core.Groups) config.ServerGroup {
+	filter := []config.ServerGroup{
+		// standard VPN
+		config.ServerGroup_STANDARD_VPN_SERVERS,
+		// P2P(the servers part of this group have from API also standard group which needs to be ignored)
+		config.ServerGroup_P2P,
```

Comment:
Will there ever by P2P servers? because Standard VPN group is checked first and all P2P have also defined.

Maybe move `config.ServerGroup_STANDARD_VPN_SERVERS` to be the last one checked, in this way if a server is part of any other specialty group will be better identified?